noxfile.py

```diff
@@ -123,6 +123,10 @@ def system(session):
     if not os.environ.get("GOOGLE_APPLICATION_CREDENTIALS", ""):
         session.skip("Credentials must be set via environment variable")
 
+    # Set test artifact bucket env var for system tests
```

Comment:
Synthtool will attempt to overwrite these lines since the `noxfile.py` is [templated](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2).

Options:
- Add a `s.replace(...)` in `synth.py` that adds in these lines ([example](https://github.com/googleapis/python-aiplatform/blob/116a29b1efcebb15bad14c3c36d3591c09ef10be/synth.py#L174))
- Move the env var setting to the top of `tests/system/aiplatform/test_dataset.py`

Reply:
Thanks for catching this, taking option 2 and moving to `tests/system/aiplatform/test_dataset.py`